applications/nrf5340_audio/src/audio/sw_codec_select.h

```diff
@@ -39,8 +39,13 @@ enum sw_codec_select {
 
 enum sw_codec_num_ch {
 	SW_CODEC_ZERO_CHANNELS,
-	SW_CODEC_MONO, /* Only use one channel */
-	SW_CODEC_STEREO, /* Use both channels */
+	SW_CODEC_ONE_CHANNEL,  /* Only use one channel */
+	SW_CODEC_TWO_CHANNELS, /* Use both channels */
+};
+
+enum sw_codec_audio_mode {
```

Comment:
We did not discuss this in the meeting. @andvib have you two discussed this naming further?

Reply:
We haven't, I think we need some input from you guys.

Would it be better if specify the mode is related to the channels? E.g. `sw_codec_channel_mode`?

Also - should we just remove `sw_codec_num_ch` since it's only being used as a number now, or should we leave it as an enum to have a guard for assigning to high numbers? If we keep it I think we must add `SW_CODEC_THREE_CHANNELS` as well

Reply:
@MariusVaardal We had a small discussion on this offline, and our suggestion is that `sw_codec_num_ch` is renamed to `sw_codec_num_instances`, and `sw_codec_audio_mode` to `sw_codec_channel_mode`

Reply:
Ok, this is not done. Discuss tomorrow.